assets/scss/style.scss

```diff
@@ -213,3 +215,106 @@ html, body {
 section.comments {
   margin-top: 2.5rem;
 }
+
+/* === All styles related to the author page, /author/author-name. === */
+
+$img-dimension: 20rem;
+$author-padding: 6rem;
+
+.main-header {
+
+}
+
+/* author cover photo */
+.author-head {
+	background-repeat: no-repeat;
+	background-position: center center;
+	background-size: contain;
+
+	height: 30rem;
+	/* TODO - just do it based on margins*/
+
+
+}
+
+
+
+/* if author has no cover photo. */
+.no-cover {
+    height: 0em;
+}
+/* Container class for author photo, bio, location, and other meta data. */
+.author-profile {
+    text-align: center;
+
+    /*
+    padding-left: $author-padding;
+    padding-right: $author-padding;
+	*/
+    margin-bottom: -4rem; /*TODO play with this value*/
+
+}
+
+
+
+
+
+
+/* head shot*/
+.author-image {
+    width: $img-dimension;
+    margin: 0 auto;
+}
+
+.author-image .img {
+    margin: 0 auto;
+    height: $img-dimension;
+    background-size: cover;
+    background-position: center center;
+    border: 1px solid $primary-color;
```

Comment:
Ok, let's keep it like that and we'll see how it looks like with other photos. We can always change it later if we want to.